packages/openactive-integration-tests/test/helpers/flow-helper.js

```diff
@@ -27,38 +37,48 @@ class FlowHelper {
   }, { cachePromiseRejection: true });
 
   C1 = pMemoize(async () => {
-    await this.getMatch();
-    if (!this.state.getMatchResponseSucceeded) throw Error('Pre-requisite step failed: opportunity feed extract failed');
+    if (this._doRunPreviousStages) {
+      await this.getMatch();
```

Comment:
Interesting... I'd understood that this was needed to ensure the stages were run in sequence? Otherwise the describe blockers weren't guaranteed to run in parallel?

Does this not force each describe block to "await" its dependencies, up the chain?